quipucords/api/management/commands/regenerate_missing_report_cache_files.py

```diff
@@ -0,0 +1,93 @@
+        """Find reports with problematic paths and return their IDs."""
+        report_ids = set()
+        reports_paths = DeploymentsReport.objects.filter(
+            status=DeploymentsReport.STATUS_COMPLETE
+        ).values("id", "cached_fingerprints_file_path", "cached_csv_file_path")
+        self.write_info(
+            f"Found {len(reports_paths)} complete DeploymentsReports to check."
+        )
+        for report_paths in reports_paths:
+            if not report_paths["cached_fingerprints_file_path"]:
+                self.write_warning(
+                    "cached_fingerprints_file_path not set for DeploymentsReport "
+                    f"{report_paths["id"]}"
+                )
+            elif not file_exists(report_paths["cached_fingerprints_file_path"]):
+                self.write_warning(
+                    f"{report_paths["cached_fingerprints_file_path"]} is missing."
+                )
+                report_ids.add(report_paths["id"])
+
+            if not report_paths["cached_csv_file_path"]:
+                self.write_warning(
+                    "cached_csv_file_path not set for DeploymentsReport "
+                    f"{report_paths["id"]}"
+                )
+            elif not file_exists(report_paths["cached_csv_file_path"]):
+                self.write_warning(
+                    f"{report_paths["cached_csv_file_path"]} is missing."
+                )
+                report_ids.add(report_paths["id"])
+
+        return report_ids
+
+    def handle(self, *args, **options):
+        """Handle this command."""
+        report_ids = self.find_reports_with_problematic_paths()
+        failed_report_ids = []
+        if report_ids:
+            report_ids = sorted(list(report_ids))
+            self.write_warning(
+                f"{len(report_ids)} DeploymentReport(s) have missing cached files."
+            )
+            for report_id in report_ids:
+                report = DeploymentsReport.objects.get(pk=report_id)
+                self.write_info(f"Rerunning fingerprint task for {report}")
+                try:
+                    report._rerun_latest_fingerprint(wait=True)
+                except Exception as e:  # noqa: BLE001
+                    self.stdout.write(self.style.ERROR(str(e)))
+                    failed_report_ids.append(report_id)
+        else:
+            self.write_success(
+                "All DeploymentReport cached_fingerprints_file_path and "
+                "cached_csv_file_path files exist."
+            )
+
+        if set(report_ids) - set(failed_report_ids):
+            successes = set(report_ids) - set(failed_report_ids)
+            self.write_success(
+                f"Successfully regnerated files for "
+                f"{len(successes)} DeploymentsReports."
+            )
+        if failed_report_ids:
+            self.write_warning(
+                f"Failed to regenerate files for "
+                f"{len(failed_report_ids)} DeploymentsReports."
```

Comment:
I don't know if that's useful, but we could also print ids of reports where we failed to regenerate cache. id is not the easiest thing to handle, but it does feel better than blank "some failed, good luck".

Reply:
Text output for this command is a lot more useful now. Here's a screenshot of the ending output from a recent run containing some successes and failures.

![command-output-example](https://github.com/user-attachments/assets/0f2e0039-b5c1-4ea2-9291-b685cb445cab)